src/NLog/LogLevel.cs

```diff
@@ -38,43 +38,43 @@ namespace NLog
     using System.ComponentModel;
 
     /// <summary>
-    /// Defines available log levels.
+    /// Defines the available log levels. They range from the lowest level <see cref="Trace"/> to highest log level <see cref="Fatal"/>.
     /// </summary>
     [TypeConverter(typeof(Attributes.LogLevelTypeConverter))]
     public sealed class LogLevel : IComparable<LogLevel>, IComparable, IEquatable<LogLevel>, IFormattable
     {
         /// <summary>
-        /// Trace log level.
+        /// 'Trace' log level. This is the 1st log level, which is the lowest log level.
         /// </summary>
         public static readonly LogLevel Trace = new LogLevel("Trace", 0);
 
         /// <summary>
-        /// Debug log level.
+        /// 'Debug' log level. This is the 2nd log level (lowest log level = <see cref="Trace"/>).
```

Comment:
Think I prefer `Debug log level (Ordinal = 1)`

Reply:
The ordinals have no relevance for users of NLog, at least not for me. The user (in this case, me) wants to know which log levels exist and what they mean. They see the list of members when they type the class name, and they put the cursor or selection at one member. And then they read `'Debug' log level`... What?? I know that it's a log level, because it's in the `LogLevel` class. I know that it's the `Debug` log level because it's named so. The current comment has absolutely no use. But when I tell the user, that it's the second log level and the lowest one is 'Trace' (see above mentioned order of log levels), then they immediately get some useful information.
You might argue against this or other enhancements with a duplication of information, but in documentations you will often find such duplications, where descriptions are repeated on different places. This is not a violation of DRY, because a) it's no code, and b) it helps the user. And if you really don't like duplicate descriptions, you might use XML doc `include` or `inheritdoc` (we use both often) to remove duplicates from the documentation where possible.

Reply:
I agree that the current xml-comment `Debug log level.` is not perfect, and more detail could be included. Ex:

- `Debug log level (Ordinal = 1) - Internal events interesting for debugging the behavior of the application`

But like the idea to improve the xml-comment for `class LogLevel` to have a xml-remark-section that lists all public loglevels ordered by severity:

```
- Trace Log Level (Ordinal = 1) - Most verbose level. Example for investigating protocol payloads during development. Almost never enabled in production
- Debug log level (Ordinal = 1) - Internal events interesting for debugging the behavior of the application
- Info log level (Ordinal = 2) - Highlight progress or lifetime events of the application
- Warn log level (Ordinal = 3) - Validation warnings or temporary failures that can be recovered
- Error Log Level (Ordinal = 4) - Exceptions or functionality has failed
- Fatal Log Level (Ordinal = 5) - Most critical level. Application is about to abort
```